src/Stratis.Bitcoin/Consensus/PeerGetHeaderRequestBehavior.cs

```diff
@@ -42,6 +42,9 @@ public sealed class RateLimitingBehavior : NetworkPeerBehavior
         /// <summary>The threshold after which the node will be banned and disconnected.</summary>
         private const int GetHeaderRequestCountThreshold = 10;
 
+        /// <summary>The threshold time span in which header requests are invalid and tracked.</summary>
+        private const int GetHeaderRequestTimestampThreshold = 10;
```

Comment:
Please add `Seconds` to the end of the variable's name. Same for the `GetHeaderRequestCountThreshold`

Reply:
Done